web-wallet/src/lib/stores/stores.d.ts

```diff
@@ -46,7 +46,7 @@ type WalletStoreServices = {
 	withdrawReward: (gasPrice: number, gasLimit: number) => Promise<any> & ReturnType<Wallet["withdrawReward"]>;
 };
 
-type WalletStore = Readable<WalletStoreContent> & WalletStoreServices;
+type WalletStore = Writable<WalletStoreContent> & WalletStoreServices;
```

Comment:
This store was meant to be read only.

Reply:
Indeed we need further discussion on this as the original `walletStore` is defined as a writable store from the beginning.
https://github.com/dusk-network/rusk/blob/b19bb89a3207c606e29d8c270859c7018af5b438/web-wallet/src/lib/stores/walletStore.js#L36

Reply:
> Indeed we need further discussion on this as the original `walletStore` is defined as a writable store from the beginning.
> 
> https://github.com/dusk-network/rusk/blob/b19bb89a3207c606e29d8c270859c7018af5b438/web-wallet/src/lib/stores/walletStore.js#L36

Yes but in my Svelte ignorance I thought I made it readable by avoiding to export `set`.

Reply:
The solution here might be a read only store which is derived into a writable one.